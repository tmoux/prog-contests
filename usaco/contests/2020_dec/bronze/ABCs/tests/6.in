110226524 28610964 724812066 696201102 835038590 138837488 806427626
