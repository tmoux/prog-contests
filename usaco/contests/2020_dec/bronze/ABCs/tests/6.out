28610964 110226524 696201102
